datagen/pom.xml

```diff
@@ -52,6 +52,12 @@
             <artifactId>spark-sql_${scala.binary.version}</artifactId>
             <version>${spark.test.version}</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.github.scopt/scopt -->
+        <dependency>
```

Comment:
Adding a dependency here makes using this a little more complicated. We either need to shade the dependency and package it with the jar, or we need to make sure that we pull in this dependency when we go to run it. If the dependency came with Spark, then it would not be a big deal because we need Spark to run, but because it does not we now need to figure out how to make that work.